### PR TITLE
osm

### PR DESCRIPTION
**Pull request by klml**
OpenStreetMap

### CODE REVIEW

File: de/software/openstreetmap.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+licensingmodel: open source
+logo: /logo/240px-Openstreetmap_logo.svg.png
+tags:
+- saas
+---
+__OpenStreetMap__ ist ein freies Projekt, das frei nutzbare Geodaten sammelt, strukturiert, diese als Open Data pflegt und Onlinelandkarten zur Verfügung stellt.
+
+---
+
+Die Landeshauptstadt München nutzt OpenStreetMap als Kartenquelle für das [Geoportal](geoportal.html) außerhalb von München, da wir dort keine amtlichen Karten vorliegen haben.
```
Comment (FabianWilms) [inline de/software/openstreetmap.md:13]:
```suggestion
Die Landeshauptstadt München nutzt OpenStreetMap als Kartenquelle für Gebiete außerhalb der Stadtgrenzen in ihrem [Geoportal](geoportal.html).
```

File: de/software/openstreetmap.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+Aber auch anderen Anwendungen wie der [Kitafinder](https://kitafinder.muenchen.de/elternportal/de/einrichtungen/karte/ergebnisse) oder ebenfalls freie [mark-a-spot](mark-a-spot.html) ([maengelmelder.muenchen.de](https://maengelmelder.muenchen.de)))nutzt OpenStreetMap.
```
Comment (FabianWilms) [inline de/software/openstreetmap.md:14]:
```suggestion
Aber auch andere freie Anwendungen wie der [Kitafinder](https://kitafinder.muenchen.de/elternportal/de/einrichtungen/karte/ergebnisse) und [mark-a-spot](mark-a-spot.html) (in Form von [maengelmelder.muenchen.de](https://maengelmelder.muenchen.de))) nutzen OpenStreetMap.
```

File: de/software/openstreetmap.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+München ist auch interessiert OpenStreetMap besser zu machen.
```
Comment (FabianWilms) [inline de/software/openstreetmap.md:16]: What do you want to imply with this sentence? It doesn't add much value here.

File: software/openstreetmap.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+licensingmodel: open source
+logo: /logo/240px-Openstreetmap_logo.svg.png
+tags:
+- saas
+---
+__OpenStreetMap__ is a free project that collects and structures freely usable geodata, maintains it as open data and provides online maps.
+
+---
+
+The City of Munich uses OpenStreetMap as a map source for the [Geoportal](geoportal.html) outside of Munich, as we do not have any official maps available there.
```
Comment (FabianWilms) [inline software/openstreetmap.md:13]:
```suggestion
The City of Munich uses OpenStreetMap as a map source for areas outside the city limits in its [Geoportal](geoportal.html).
```

File: software/openstreetmap.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+But other applications such as the [Kitafinder](https://kitafinder.muenchen.de/elternportal/de/einrichtungen/karte/ergebnisse) or the free [mark-a-spot](mark-a-spot.html) ([maengelmelder.muenchen.de](https://maengelmelder.muenchen.de)) also use OpenStreetMap.
```
Comment (FabianWilms) [inline software/openstreetmap.md:14]:
```suggestion
But other free applications such as [Kitafinder](https://kitafinder.muenchen.de/elternportal/de/einrichtungen/karte/ergebnisse) and [mark-a-spot](mark-a-spot.html) (in the form of [maengelmelder.muenchen.de](https://maengelmelder.muenchen.de)) also use OpenStreetMap.
```

**Review by FabianWilms — CHANGES_REQUESTED** [review]

**Review by FabianWilms — APPROVED** [review]

File: de/software/openstreetmap.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+title: OpenStreetMap
+developerlink: https://openstreetmap.org
+licensingmodel: open source
+logo: /logo/240px-Openstreetmap_logo.svg.png
+tags:
+- saas
```
[the remark this answers is not in the document]
Comment (klml) [reply de/software/openstreetmap.md:7]: there is no [Cooperation label](https://github.com/it-at-m/opensource.muenchen.de/blob/main/.vitepress/theme/enhancements/i18n/lang/de.ts), yet ;)

[the remark this answers is not in the document]
Comment (FabianWilms) [reply de/software/openstreetmap.md:7]: https://github.com/it-at-m/opensource.muenchen.de/blob/main/.vitepress/theme/enhancements/i18n/lang/de.ts#L9

**Pull request merged**